src/praisonai-agents/tests/unit/tools/test_send_message_tool.py

```
@@ -0,0 +1,128 @@
+            }
+        ]
+    finally:
+        clear_outbound_messenger(token)
+
+
+def test_media_directive_is_parsed():
+    messenger = FakeMessenger()
+    token = register_outbound_messenger(messenger)
+    try:
+        send_message("origin", "Report ready MEDIA:/tmp/report.pdf")
+        target, text, media = messenger.sent[0]
```

Ruff (0.15.18)

[warning] Unpacked variable `target` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)